### PR TITLE
Isochron adding isochron page

### PR DESCRIPTION
**Pull request by ZoeBrunet**
Add a new page to view isochrons on a map

### CODE REVIEW

File: isochron.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  </div>
+  <script type="text/javascript" src="./assets/jquery/jquery-1.11.0.min.js"></script>
+  <script type="text/javascript" src="./assets/jquery-ui.js_1.11.4/jquery-ui.js"></script>
+  <script type="text/javascript" src="./assets/leaflet_0.7.7/leaflet.js"></script>
+  <script type="text/javascript" src="./assets/js/shortcut.js"></script>
+  <script type="text/javascript" src="./assets/js/menu.js"></script>
+  <script type="text/javascript" src="./assets/js/lib.js"></script>
+  <script type="text/javascript" src="./assets/js/conversion.js"></script>
+  <script type="text/javascript" src="./assets/js/isochron.js"></script>
+  <script type="text/javascript" src="./assets/js/slider.js"></script>
```
Comment (nlehuby) [inline isochron.html]: this file does not exist.

File: isochron.html [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+            <td width="310px">
+                <label for="from_text">Départ : </label>
+                <input id="from_text" name="from_text" type="text">      <input id="from" name="from" type="hidden">
+              </td>
+              <td  width="300px" valign="top">
+                date : <input type="texte" id="date" name="date" size="5"><br>
+                heure : <input type="texte" id="time" name="time" size="1"><br>
+              </td>
+              <td  width="370px">
+                durée min : <input type="texte" id="min_duration" name="min_duration" size="2" readonly style="border:0; color:#f6931f; font-weight:bold;">
```
Comment (nlehuby) [inline isochron.html]: maybe add a unit of time for the min/max duration display

File: assets/js/isochron.js [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+      $( "#min_duration" ).val(ui.values[0]);
+      $( "#max_duration" ).val(ui.values[1]);
+    }
+  });
+  $( "#min_duration" ).val( "$" + $( "#slider-range" ).slider("values", 0 ));
+  $( "#max_duration" ).val( "$" + $( "#slider-range" ).slider("values", 1 ));
+});
+
+$(function() {
+  $( "#date" ).datepicker();
```
Comment (nlehuby) [inline assets/js/isochron.js]: :+1:

Comment (nlehuby) [inline assets/js/isochron.js]: the date is expected as DD/MM/YYYY. 
Using the datepicker, we get MM/DD/YYYY :bug:

**Comment by ZoeBrunet** [comment]
@nlehuby the corrections has been pushed

**Comment by nlehuby** [comment]
Unless the isochrons API works out of coverage (with <navitia_url>/v1/isochrons instead of <navitia_url>/v1/coverage/<my_coverage>/isochrons), you should remove the metasystem feature.

**Pull request merged**